### PR TITLE
Document msgs certificate

### PR DESCRIPTION
**Pull request by oded282**

### CODE REVIEW

File: bftengine/src/bftengine/messages/MsgsCertificate.hpp [hunk 1]
```diff
@@ -166,7 +175,7 @@ void MsgsCertificate<T, SelfTrust, SelfIsRequired, KeepAllMsgs, ExternalFunc>::a
 
     if (!ExternalFunc::equivalent(representativeMsg, msg)) return;  // msg should be ignored
 
-    relevantClass = 0;
+    relevantClass = 0;  // the class of the self message
```
Comment (teoparvanov) [inline bftengine/src/bftengine/messages/MsgsCertificate.hpp:178]: @oded282 can you explain what this "class" means? it's not very obvious...
Thanks!

Comment (oded282) [reply bftengine/src/bftengine/messages/MsgsCertificate.hpp:182]: Sure.

**Comment by andrewjstone** [comment]
So nice to see this. Thank you Oded!

**Comment by HristoStaykov** [comment]
LGTM. Thank you Oded.

**Pull request merged**